packages/webapp/pages/posts/[id].tsx

```diff
@@ -65,6 +66,15 @@ const ShareNewCommentPopup = dynamic(
 );
 const Custom404 = dynamic(() => import('../404'));
 
+export const getSeoDescription = (post: Post): string => {
+  if (post?.summary?.length > 0) {
+    return post?.summary;
+  }
+  if (post?.description) {
```

Comment:
Why don't we check for length here?
Could you try with a empty description object and see what happens?